website/docs/examples/cxf/soap-client.md

```
@@ -0,0 +1,146 @@
+cd ../soap-server && camel run *
+```
+
+## Running
+
+```bash
+camel run *
+```
+
+You should see:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)